tests/test_helpers.py

```diff
@@ -1,27 +1,55 @@
 """Tests for helper functions."""
 
 from __future__ import print_function, unicode_literals
-from praw.helpers import chunk_sequence
-from .helper import PRAWTest
+from praw import helpers, objects
+from .helper import PRAWTest, betamax
 
 
-class HeleperTest(PRAWTest):
+class HelperTest(PRAWTest):
+    def setUp(self):
+        super(HelperTest, self).setUp()
+        self.verbosity = 0
+
+    def test_bounded_set_fifo_unique(self):
+        bounded_set = helpers.BoundedSet(20)
+        for i in range(10):
+            bounded_set.add(i)
+        old_fifo = bounded_set._fifo[:]
+        for n in range(10):
+            bounded_set.add(n)
+        self.assertEqual(old_fifo, bounded_set._fifo)
+
     def test_chunk_sequence(self):
         s = 'abcdefgh'
-        self.assertEqual(chunk_sequence(s, 4, allow_incomplete=True),
+        self.assertEqual(helpers.chunk_sequence(s, 4, allow_incomplete=True),
                          ['abcd', 'efgh'])
-        self.assertEqual(chunk_sequence(s, 3, allow_incomplete=True),
+        self.assertEqual(helpers.chunk_sequence(s, 3, allow_incomplete=True),
                          ['abc', 'def', 'gh'])
-        self.assertEqual(chunk_sequence(s, 3, allow_incomplete=False),
+        self.assertEqual(helpers.chunk_sequence(s, 3, allow_incomplete=False),
                          ['abc', 'def'])
-        self.assertEqual(chunk_sequence(s, 10, allow_incomplete=True),
+        self.assertEqual(helpers.chunk_sequence(s, 10, allow_incomplete=True),
                          ['abcdefgh'])
-        self.assertEqual(chunk_sequence(s, 10, allow_incomplete=False),
+        self.assertEqual(helpers.chunk_sequence(s, 10,
+                                                allow_incomplete=False),
                          [])
 
         # Returned sequence is same type as input
         s = ['a', 'b', 'c']
-        self.assertEqual(chunk_sequence(s, 2, allow_incomplete=True),
+        self.assertEqual(helpers.chunk_sequence(s, 2, allow_incomplete=True),
                          [['a', 'b'], ['c']])
-        self.assertEqual(chunk_sequence(s, 2, allow_incomplete=False),
+        self.assertEqual(helpers.chunk_sequence(s, 2, allow_incomplete=False),
                          [['a', 'b']])
+
+    def test_id_to_id36(self):
+        alphabet = '0123456789abcdefghijklmnopqrstuvwxyz'
+        alphabet_len = len(alphabet)
+        for i in range(alphabet_len):
+            self.assertEqual(alphabet.index(
+                             helpers.convert_numeric_id_to_id36(i)), i)
+
+    @betamax()
+    def test_comment_stream(self):
+        generator = helpers.comment_stream(self.r, "all",
+                                           verbosity=self.verbosity)
+        for i in range(10001):
```

Comment:
This number was arbitrary-- trying to find the minimal value which would cover all lines except for the "same response found" lines and the except block in the underlying `_stream_generator` was taking forever as it could change each time. I used a for loop instead of passing a limit to the generator because it's more memory efficient this way (my vm crashed the first time using `list` and I was afraid to do it again, even if coincidental)